src/interactive/scala/tools/nsc/interactive/Global.scala

```diff
@@ -1175,9 +1176,12 @@ class Global(settings: Settings, _reporter: Reporter, projectName: String = "")
     }
   }
   object CompletionResult {
+    // outer reference thing (twice) is https://github.com/scala/bug/issues/11534 ?
+    @nowarn("cat=unchecked&msg=The outer reference in this type test cannot be checked at run time")
```

Comment:
The other lint-related PR applies this in common settings. "Saturday night special."

Reply:
Yeah and I don't know why it didn't take, here. I'll dig into it, at least a little.

Reply:
after digging:

the thing you put in common settings was `:ws` (summarize warning). I don't think that's what we want, because we really want to either fix these or silence them, not get "2 unchecked warnings; re-run with -unchecked for details". unchecked warnings can be Very Bad Indeed, so "2 unchecked warnings" is, or should be, an alarming thing to see in the build log

I'm not sure if you really _meant_ `:ws`, or whether you confused **s**ummarize with **s**ilence, but in any case, I'll change it to `:s`. it might be preferable to actually audit the individual sites and `@nowarn` them on a case-by-case basis, but I leave that as “future work”, as the academics say